docs/embedding/start.md

```diff
@@ -14,42 +14,42 @@ What is embedding, and how does it work?
 
 Embed individual dashboards, questions, or the query builder in your app with an interactive wizard and simple drop-in script, with minimal or no coding required. Control component UI and theming. Integrate your app's auth with Metabase SSO.
 
-## [Modular embedding SDK](./sdk/introduction.md)
+### [Modular embedding SDK](./sdk/introduction.md)
 
 With the Modular embedding SDK, you can embed individual Metabase components with React (like standalone charts, dashboards, the query builder, and more). You can manage access and interactivity per component, and you have advanced customization for seamless styling.
 
-## [Modular embedding SDK quickstart](./sdk/quickstart.md)
+### [Modular embedding SDK quickstart](./sdk/quickstart.md)
 
 Jump to a SDK quickstart with a sample React application.
 
-## [Static embedding](./static-embedding.md)
+### [Guest embedding](./guest-embedding.md)
 
-Also known as Signed Embedding, Static embedding is a secure way to embed charts and dashboards. Static embeds are view only; people won't be able to drill-through charts and tables.
+Also known as Signed Embedding, guest embedding is a secure way to embed charts and dashboards. Guest embeds are view only; people won't be able to drill-through charts and tables.
```

Comment:
we can probably remove the mention of signed embedding at this point, been long enough